exampleplugins/Frontend/SwagService/Bootstrap.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * The Bootstrap class is the main entry point of any shopware plugin.
+ *
+ * Short function reference
+ * - install: Called a single time during (re)installation. Here you can trigger install-time actions like
+ *   - creating the menu
+ *   - creating attributes
+ *   - creating database tables
+ *   You need to return "true" or array('success' => true, 'invalidateCache' => array()) in order to let the installation
+ *   be successfull
+ *
+ * - update: Triggered when the user updates the plugin. You will get passes the former version of the plugin as param
+ *   In order to let the update be successful, return "true"
+ *
+ * - uninstall: Triggered when the plugin is reinstalled or uninstalled. Clean up your tables here.
+ */
+class Shopware_Plugins_Frontend_SwagService_Bootstrap extends Shopware_Components_Plugin_Bootstrap
+{
+    /**
+     * Register the namespace of our plugin globally
+     */
+    public function afterInit()
+    {
+        $this->Application()->Loader()->registerNamespace(
+            'Shopware\SwagService',
+            $this->Path()
+        );
+    }
+
+    /**
+     * Return the version of the plugin.
+     *
+     * @return mixed
+     * @throws Exception
+     */
+    public function getVersion()
+    {
+        return '1.0.0';
+    }
+
+    /**
+     * Return the label of the plugin
+     *
+     * @return string
+     */
+    public function getLabel()
+    {
+        return 'SwagService';
+    }
+
+    /**
+     * Register Service + an example controller PreDispatch method
+     *
+     * @return bool
+     */
+    public function install()
+    {
+        $this->subscribeEvent(
+            'Enlight_Bootstrap_InitResource_swag_service_plugin.tax_calculator',
+            'onInitTaxCalculator'
+        );
+
+        $this->subscribeEvent(
+            'Enlight_Controller_Action_PreDispatch_Frontend',
+            'onFrontendPreDispatch'
+        );
+
+        return true;
+    }
+
+    /**
+     * Return an instance of our new service class
+     *
+     * @return \Shopware\SwagService\Component\TaxCalculator
+     */
+    public function onInitTaxCalculator()
+    {
+        return new \Shopware\SwagService\Component\TaxCalculator(
+            $this->get('corelogger')
```

Comment:
We should use `pluginlogger` here. The `corelogger` should be reserved for logging within the shopware core.